package.json

```
@@ -7,6 +7,33 @@
     "dist",
     "src/tokens"
   ],
+  "exports": {
+    "./themes/light.css": "./dist/css/functional/themes/light.css",
+    "./themes/light-high-contrast.css": "./dist/css/functional/themes/light-high-contrast.css",
+    "./themes/light-colorblind.css": "./dist/css/functional/themes/light-colorblind.css",
+    "./themes/light-colorblind-high-contrast.css": "./dist/css/functional/themes/light-colorblind-high-contrast.css",
+    "./themes/light-tritanopia.css": "./dist/css/functional/themes/light-tritanopia.css",
+    "./themes/light-tritanopia-high-contrast.css": "./dist/css/functional/themes/light-tritanopia-high-contrast.css",
+    "./themes/dark.css": "./dist/css/functional/themes/dark.css",
+    "./themes/dark-high-contrast.css": "./dist/css/functional/themes/dark-high-contrast.css",
+    "./themes/dark-dimmed.css": "./dist/css/functional/themes/dark-dimmed.css",
+    "./themes/dark-dimmed-high-contrast.css": "./dist/css/functional/themes/dark-dimmed-high-contrast.css",
+    "./themes/dark-colorblind.css": "./dist/css/functional/themes/dark-colorblind.css",
+    "./themes/dark-colorblind-high-contrast.css": "./dist/css/functional/themes/dark-colorblind-high-contrast.css",
+    "./themes/dark-tritanopia.css": "./dist/css/functional/themes/dark-tritanopia.css",
+    "./themes/dark-tritanopia-high-contrast.css": "./dist/css/functional/themes/dark-tritanopia-high-contrast.css",
```

If you feel okay matching over everything in `dist/css/functional/themes/*` we could update this to be:


```suggestion
    "./themes/*.css": "./dist/css/functional/themes/*.css",
```

Same idea could be applied to the other ones too if it would be helpful. Also totally understand wanting it to be explicit too 👍